package.json

```diff
@@ -52,7 +52,7 @@
   },
   "main": "dist/vue-analytics-facebook-pixel.js",
   "repository": {
-    "url": "https://github.com/wemake-services/vue-analytics-facebook-pixel.git",
+    "url": "https://github.com/hex-digital/vue-analytics-facebook-pixel.git",
```

Comment:
Can you please revert this change?

Reply:
Ah! Thank you. This was a mistake of me using my package temporarily in the mean time.